scripts/microos

```diff
@@ -54,11 +55,17 @@ def update_group(group):
 
 
 def read_and_parse(url):
-    with urllib.request.urlopen(url) as resp:
-        if url.endswith(".gz"):
-            return ET.parse(gzip.open(resp)).getroot()
-        else:
-            return ET.parse(resp).getroot()
+    if url.endswith(".zst"):
+        os.system(f"wget -O primary.xml.zst {url} && unzstd --rm primary.xml.zst")
```

Comment:
I think that the correct what is not to call `wget` nor `unzstd`. The python code is already downloading the `primary.xml.zst` with `urllib.request.urlopen()`. We can add a new dependency: [zstandard](https://pypi.org/project/zstandard/) and use it to decompress it